cfme/infrastructure/provider/__init__.py

```diff
@@ -248,6 +199,9 @@ def get_clusters(self):
             web_clusters.append(Cluster(icon.name, self))
         return web_clusters
 
+    def as_fill_value(self):
```

Comment:
this is one of widgetastic features. If you put object to fill method. fill will automatically pick up value from this method.